test/fixtures/poro.rb

```diff
@@ -38,9 +38,16 @@ class ProfilePreviewSerializer < ActiveModel::Serializer
 
 Post     = Class.new(Model)
 Like     = Class.new(Model)
-Author   = Class.new(Model)
+Author   = Class.new(Model) do
+  # To confirm cache_key is set using updated_at and cache_key option passed to cache
+  undef_method :cache_key
+end
 Bio      = Class.new(Model)
 Blog     = Class.new(Model)
+Article  = Class.new(Model) do
+  # To confirm error is raised when cache_key is not set and cache_key option not passed to cache
+  undef_method :cache_key
+end
```

Comment:
Why wouldn't you prefer for all the objects to be defined in a single location given the existence of this file.  It seems to me that if you want the test using these objects to create the objects they will test, then all tests should do the same and eliminate the need for this file.  Is that the direction you're going?